lib/src/model/stripe_account_link_request.dart

```diff
@@ -0,0 +1,8 @@
+class StripeAccountLinkRequest {
```

Comment:
Thats a wallet specific file why we need it in sdk?

Reply:
i wasn't sure if there was a use case where it'd matter or not! perfectly fine w/ deleting all the stripe requests